tests/appsec/iast/source/test_path.py

```diff
@@ -0,0 +1,26 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import coverage
+from .._test_iast_fixtures import BaseSourceTest
+
+
+@coverage.basic
+class TestPath(BaseSourceTest):
+    """Verify that request path is tainted"""
+
+    endpoint = "/iast/source/path/test"
+    source_type = "http.request.path"
+    source_name = None
+    source_value = "/iast/source/path/test"
+    requests_kwargs = [{"method": "GET"}]
+
+    def test_source_reported(self):
```

Comment:
You can remove these 6 lines. It's useful only if you need to decorate them. It you don't have any decortaor, just remove them, it'll inherits them from `BaseSourceTest`

Reply:
Good point, removed.